### PR TITLE
Oai-Controller Changes for O1-Telnet

### PR DESCRIPTION
**Pull request by jain-ashish-sam**

### CODE REVIEW

File: internal/controller/resources_cucp.go [hunk 1]
```diff
@@ -218,13 +218,16 @@ func (resource CuCpResources) GetDeployment(log logr.Logger, ranDeployment *work
 								corev1.EnvVar{
 									Name:  "USE_ADDITIONAL_OPTIONS",
 									Value: "--sa --log_config.global_log_options level,nocolor,time",
+									// " --gNBs.[0].E1_INTERFACE.[0].ipv4_cucp 192.168.77.2" +
```
Comment (arora-sagar) [inline internal/controller/resources_cucp.go]: @jain-ashish-sam you can remove the comments, they are not relevant in this use case.

File: internal/controller/resources_cuup.go [hunk 2]
```diff
@@ -111,7 +111,11 @@ func (resource CuUpResources) GetDeployment(log logr.Logger, ranDeployment *work
 								},
 								corev1.EnvVar{
 									Name:  "USE_ADDITIONAL_OPTIONS",
-									Value: "--sa",
+									Value: "--sa --log_config.global_log_options level,nocolor,time",
+									// 		" --gNBs.[0].E1_INTERFACE.[0].ipv4_cucp 192.168.77.2" +
```
Comment (arora-sagar) [inline internal/controller/resources_cuup.go]: @jain-ashish-sam same here

File: internal/controller/resources_du.go [hunk 3]
```diff
@@ -178,25 +177,22 @@ func (resource DuResources) GetDeployment(log logr.Logger, ranDeployment *worklo
...
+										" --telnetsrv --telnetsrv.shrmod o1 --telnetsrv.listenaddr 192.168.74.2",
 								},
 								corev1.EnvVar{
-									Name:  "USE_ADDITIONAL_OPTIONS",
-									Value: "--sa --rfsim --log_config.global_log_options level,nocolor,time",
-								},
-								corev1.EnvVar{
-									Name:  "USE_VOLUMED_CONF",
-									Value: "yes",
+									Name:  "ASAN_OPTIONS",
```
Comment (arora-sagar) [inline internal/controller/resources_du.go]: @jain-ashish-sam Not needed

**Comment by arora-sagar** [comment]
@jain-ashish-sam Did you test the code? Can you explain how you did the testing?

**Comment by jain-ashish-sam** [comment]
I have created the ReadME for testing, Please refer [here](https://github.com/jain-ashish-sam/oai/blob/main/test-infra/ReadMe.md)

**Pull request merged**